en/docs/guides/passwordless/hypr.md

```diff
@@ -0,0 +1,84 @@
+# Passwordless login with HYPR
+
+[HYPR](https://www.hypr.com/) is a passwordless authentication method that allows users to log in to applications using biometrics on their devices. As a FIDO-certified platform, HYPR eliminates risks prevalent in password-based authentication, such as phishing and password reuse.
+
+This guide explains how you can use HYPR to add passwordless login to applications registered in the Identity Server.
+
+## Prerequisites
+
+- You need to configure the HYPR environment and have access to the HYPR control center. Learn how to do it in the [HYPR documentation](https://docs.hypr.com/hyprcloud/docs/cc-std).
+- You need to [set up the sample]({{base_path}}/guides/adaptive-auth/adaptive-auth-overview/#set-up-the-sample) application.
```

Comment:
The use of a SAML application could lead to the perception that this connector is exclusively compatible with apps based on SAML protocols. Therefore, I suggest including a note to clarify that this connector is also compatible with applications based on OIDC protocols.

Reply:
Shall we add it like this :arrow_down: 

`You need to set up your application. For this tutorial, we will be using the [sample SAML application]({{base_path}}/guides/adaptive-auth/adaptive-auth-overview/#set-up-the-sample) application. Note that you can use any application you wish to use.`